src/dodal/beamline_specific_utils/i03.py

```diff
@@ -0,0 +1,16 @@
+from dataclasses import dataclass
+
+from dodal.devices.aperturescatterguard import SingleAperturePosition
+
+I03_BEAM_HEIGHT = 20
+
+
+@dataclass
+class BeamSize:
+    x_um: float | None
+    y_um: float | None
+
+
+def beam_size_from_aperture(position: SingleAperturePosition):
+    aperture_size = position.radius_microns
+    return BeamSize(aperture_size, I03_BEAM_HEIGHT if aperture_size else None)
```

Comment:
I don't think this file has to be an i03-specific util. We could read the `I03_BEAM_HEIGHT` from some parameter and then have this as a generic util

Reply:
yeah, I kind of agree, there will be more beamlines that could use this, but I think it's easy enough to move it when we need it